src/Aserto.AspNetCore.Middleware/Clients/Directory/V3/Directory.cs

```diff
@@ -102,12 +103,18 @@ public Directory(IOptions<AsertoDirectoryOptions> options, ILoggerFactory logger
         /// </summary>
         /// <param name="type">The type of the object.</param>
         /// <param name="id">The id of the object.</param>
+        /// <param name="displayName">The display name of the object.</param>
+        /// <param name="properties">A struct representing the properties bag of the object.</param>
+        /// <param name="hash">The hash of the object.</param>
         /// <returns>A new <see cref="Object"/>.</returns>
-        public static Object BuildObject(string type, string id)
+        public static Object BuildObject(string type, string id, string displayName = "", Struct properties = null, string hash = "")
```

Comment:
Should the last param be called `etag`? I think we dropped the `hash` keyword in favour of `etag` in v3/

Reply:
yeah, it really should bee tag.